.github/workflows/ci.yml

```diff
@@ -62,22 +65,32 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm install sequelize@${{ matrix.sequelize-version }}
-      - run: npm run build
-      - run: npm run test
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn add sequelize@${{ matrix.sequelize-version }} --ignore-engines
+      - run: yarn test
   release:
     name: Release
     runs-on: ubuntu-latest
-    needs: [test-sqlite, test-postgres, test-mysql]
-    if: github.event_name == 'push' && github.ref == 'refs/heads/release'
+    needs:
+      [
+        test-sqlite,
+        test-postgres,
+        test-mysql
+      ]
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/v6' || github.ref == 'refs/heads/v7')
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12.x
-      - run: npm install
+          node-version: 16.x
+      - run: yarn install --frozen-lockfile
       - run: npx semantic-release
+      - id: sequelize
```

Comment:
rename to `cli` here?

Reply:
I thought this id was internal so not really necessary, but I can change it if you want

Reply:
True. Fine with me